04EstruturasDeControle/test/EstruturasDeControleExercicioTest.java

```diff
@@ -1,8 +1,62 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
 public class EstruturasDeControleExercicioTest {
     // - Funcionalidade - Retornar nome do mês do ano para qualquer número
     // do mês passado. Caso seja um número inválido, retornar message de erro
 
-    /* -----------------
-     * TODO
-     * ----------------- */
+    String meses(int numeroMes) {
+        String meses;
+
+        if (numeroMes == 1) {
+            meses = "Janeiro";
+        } else if (numeroMes == 2) {
+            meses = "Fevereiro";
+        } else if (numeroMes == 3) {
+            meses = "Março";
+        } else if (numeroMes == 4) {
+            meses = "Abril";
+        } else if (numeroMes == 5) {
+            meses = "Maio";
+        } else if (numeroMes == 6) {
+            meses = "Junho";
+        } else if (numeroMes == 7) {
+            meses = "Julho";
+        } else if (numeroMes == 8) {
+            meses = "Agosto";
+        } else if (numeroMes == 9) {
+            meses = "Setembro";
+        } else if (numeroMes == 10) {
+            meses = "Outubro";
+        } else if (numeroMes == 11) {
+            meses = "Novembro";
+        } else if (numeroMes == 12) {
+            meses = "Dezembro";
+        } else {
+            meses = "O número não corresponde ao código.";
+        }
+        return meses;
+
+    }
+
+    @Test
+    void resultado() {
```

Comment:
[Opicional]

É interessante melhorar o nome do método para algum que faça mais sentido para o que está sendo testado. Um exemplo abaixo:
```suggestion
    void mesValidoEInvalido() {
```